demo/src/app/components/dropdown/demos/basic/dropdown-basic.html

```diff
@@ -0,0 +1,8 @@
+ <div ngbDropdown [open]="true">
+    <ngbd-button class="btn btn-primary" ngbDropdownToggle>Toggle dropdown</ngbd-button>
+    <div class="dropdown-menu" aria-labelledby="dropdownMenu1">
```

Comment:
there is a missing `id="dropdownMenu1"` on one of the elements (don't have BS markup in front of me to see where exactly it needs to be added)